### ISSUE TITLE
Can RTKPOST get a comprehensive solution for all epochs of the baseline？

### ISSUE THREAD

**Issue by geovey** [tomojitakasu/RTKLIB#615]
Sincerely consult everyone, I want to get the comprehensive results of the baseline through rtklib, rather than the results of each epoch; For example, one hour observation data can obtain an accurate baseline length;

Can someone help me?

**Comment by DavidKelleySCSC** [comment]
I am sorry but I not not follow you here.  Are you simply asking for an average of the 'fixed' resulting positional estimates of such a time period?  In this case just pipe the outputs to a file an process that.  Keep in mind the accuracy of the baseline depends on the RTK process, but the accuracy of the initial base location does not and may itself contain offset errors.

**Comment by geovey** [comment]
Thank you for your answer. I think it is not as simple as obtaining the average value of each ephemera, which is similar to commercial GNSS software (such as LGO and TBC). The observation data of all ephemera should be integrated to carry out unified adjustment!

If the average value is simply used, there will be a very large deviation (multi-path and cycle slips make the positioning result have a very large deviation).

**Comment by akstuki** [comment]
I agree that once the single-epoch solution is fixed, we could average the results to get the static comprehensive result.

for deviation, that's not caused by the method we used here (the single alo based), even we use other method, multi-path is also in.

cycle slips is another thing, if we use other method, of course we could fix them first, then calculate. but I believe that does not make big difference. this is my guess.
